backend/backend/api/routes.py

```diff
@@ -293,6 +295,11 @@ async def github_comment(repo, pr_number, request):
     return response
```

Comment:
Inconsistent code: github_comment and github_pr functions have different return types.